.gitignore

```diff
@@ -12,3 +12,4 @@ Gemfile.lock
 spec/examples.txt
 .idea
 .ruby-version
+.ruby-gemset
```

Comment:
This feels super archaic. But ok.

Reply:
Even some rbenv users use the gemset plugin to make it less awkward to work on (in my case dozens) or projects on the same machine. It comes in handy.